frontends/web/src/routes/device/settings/components/deviceinfo.jsx

```diff
@@ -28,15 +28,15 @@ export default class DeviceInfo extends Component {
         super(props);
         this.state = {
             active: false,
-            deviceInfo: 0,
+            deviceInfo: '',
```

Comment:
better to not mix string type with object type. init like deviceInfo: undefined.